### PR TITLE
Add keymap preference

### LINKED ISSUE #183: Add preference for keymap
As a user that uses different keyboard templates there should be an option to change the keyboard layout for keyboard shortcuts.

### PR DESCRIPTION
**Pull request by sirkrypt0**
Closes #183

### CODE REVIEW

**Comment by frcroth** [comment]
Can be merged IMO

**Comment by sirkrypt0** [comment]
should have tests

**Comment by frcroth** [comment]
> 
> 
> should have tests

Are you working on this?

**Review by oleschl — APPROVED** [review]

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/setUp.st [hunk 1]
```diff
@@ -0,0 +1,5 @@
+resources
```
Comment (frcroth) [inline packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/setUp.st]: Change to running

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/structureKeymapPreference.st [hunk 2]
```diff
@@ -0,0 +1,4 @@
+constants
+structureKeymapPreference
+
+	^ (self textEditor class asString , '>>', 'structureKeymap') asSymbol
```
Comment (frcroth) [inline packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/structureKeymapPreference.st:4]: What's that?

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/tearDown.st [hunk 3]
```diff
@@ -0,0 +1,5 @@
+resources
```
Comment (frcroth) [inline packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/tearDown.st]: running

**Review by frcroth — CHANGES_REQUESTED** [review]

**Review by frcroth — APPROVED** [review]

**Pull request merged**